### PR TITLE
Build multiarch packages

### PR DESCRIPTION
**Pull request by jakirkham**
Configure the feedstock to build and publish multiarch packages.

cc @jayfurmanek

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @jayfurmanek

**Comment by jakirkham** [comment]
The `aarch64` builds are failing as `rust` packages have not been produced yet. Please see issue ( https://github.com/conda-forge/rust-feedstock/issues/43 ) for details.

**Comment by jakirkham** [comment]
Have restarted the failing checks now that the remaining `rust` packages have been published. Let's see if that fixes things. 🤞 🍀

**Comment by jakirkham** [comment]
Looks like it passes! 🎉 Ready for review, @dbast 😄

**Comment by dbast** [comment]
Wow. Nice! Thanks! That means also the conda-build run requirements can be changed :)

**Comment by jakirkham** [comment]
Of course! Thanks also for the review. 😀

Good point! Would you like to raise an issue or PR to conda-build? 😉

**Comment by dbast** [comment]
Done https://github.com/conda-forge/conda-build-feedstock/pull/115 :)

File: conda-forge.yml [hunk 1]
```diff
@@ -5,4 +5,6 @@ appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```
Comment (isuruf) [inline conda-forge.yml]: Why use azure?

Comment (dbast) [reply conda-forge.yml]: What to use instead? 
```
  linux_aarch64: native
  linux_ppc64le: default
```

Is this about build speed and distributing the CI load to different providers?

Comment (isuruf) [reply conda-forge.yml]: Use,
```
  linux_aarch64: default
  linux_ppc64le: default
```

azure runs on x86_64 hardware and uses QEMU. drone and travis-ci runs on native hardware and therefore will have no bugs due to QEMU. For example, in numpy, there were tests failures due to QEMU.

**Comment by jakirkham** [comment]
@isuruf , I’ve just copied what I have found in feedstocks updated by the migrater. If we are wanting to change the standard, shouldn’t we also update the migrater?

**Comment by jakirkham** [comment]
In any event this is sounding like a discussion that should happen somewhere other than this specific feedstock. Could you please open an issue on this broader point?

**Comment by isuruf** [comment]
> shouldn’t we also update the migrater?

We have updated it. You are copying configurations from older feedstocks

**Comment by dbast** [comment]
ok. I'll then update ripgrep once again (even though the builds look fine) so nobody can copy some non-standard configuration to other feedstocks from it :)

**Comment by jakirkham** [comment]
@isuruf, is this documented somewhere? It’s difficult to tell what we should be doing if things are changing midstream.

**Comment by jakirkham** [comment]
@isuruf, I’ve opened issue ( https://github.com/conda-forge/conda-forge.github.io/issues/870 ) so we can firm things up a bit with ppc64le/aarch64 best practices (assuming that is needed) and improve the info sharing process here.

**Pull request merged**